abbr/utils.py

```diff
@@ -36,7 +36,8 @@ def get_random_name():
 
 
 def get_expiry():
-    return datetime.now() + app.config.get('EXPIRY')
+    # Create default expiry and add an additional second for processing time
+    return datetime.now() + app.config.get('EXPIRY') + timedelta(seconds=1)
```

Comment:
I'm not really happy with this. I'd prefer to either find a library that is capable of rounding up or adding the second for display purposes only.

In the second case, the method `get_expiry` could return a tuple of a time and a description.